irsa/helm/irsa/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: irsa
-version: 1.7.2
+version: 1.7.4
```

Comment:
I'm keeping the chart version the same as the release version in domsed as well, though it's probably not the best practice - that said the chart always changes with the releases too, but something to think about dont copy my bad processes :-)